test/compilable/test18905.d

```diff
@@ -0,0 +1,3 @@
+// https://issues.dlang.org/show_bug.cgi?id=18905
```

Comment:
Don't you need -betterC compiler flags here?  Also, you might want to add this to the existing betterc.d test file.

Reply:
Yes, good catch!